stacks/udpconn.go

```diff
@@ -0,0 +1,230 @@
+package stacks
```

Comment:
please clean up comments in this file. there seems to be lots of comments left over from a copy-pase operation.